server/FeatureExtraction/FeatureExtraction.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+import pandas
+import scipy
+import skimage.io
+import skimage.morphology
+
+import histomicstk as htk
+from ctk_cli import CLIArgumentParser
+
+import logging
+logging.basicConfig()
+
+
+def main(args):
+
+    # Read Input Image
+    print('>> Reading input image')
+
+    print '--- Image Name = ', args.inputImageFile
+
+    Standard = skimage.io.imread(args.inputImageFile)[:, :, :3]
+    print '--- Original Image Size = ', Standard.shape
+
+    # Reduce image size for testing
+    Standard = Standard[50:200, 50:200]
```

Comment:
@slee172 Remove this hard coded cropping. Run it on small image files for now. You can always crop a large image outside, save the cropped image, and use that for testing.